### PR TITLE
Update _afk.js

### PR DESCRIPTION
**Pull request by Arya274**
Tambahin design text dan nama agar diliat bagus:v

### CODE REVIEW

File: plugins/_afk.js [hunk 1]
```diff
@@ -12,12 +12,19 @@ Anda berhenti AFK
...
     if (!user) continue
+    let name = m.fromMe ? conn.user : conn.contacts[m.sender]
     let afkTime = user.afk
     if (!afkTime || afkTime < 0) continue
     let reason = user.afkReason || 'AFK'
     m.reply(`
-Jangan tag dia, dia lagi ${reason}
+Ssst orangnya lagi AFK
+
+*Username :* ${name.vnmae || name.notify || name.name || ('+' + name.jid.split`@`[0])}
```
Comment (Nurutomo) [inline plugins/_afk.js:22]: this.getName(jid)

File: plugins/_afk.js [hunk 2]
```diff
@@ -12,12 +12,19 @@ Anda berhenti AFK
...
+*Alasan :* ${reason}
 Selama ${clockString(new Date - afkTime)}
+
+Follow NOW:
+@Nurutomo
```
Comment (Nurutomo) [inline plugins/_afk.js:27]: Uhh

**Review by Nurutomo — COMMENTED** [review]
Hmm

File: plugins/_afk.js (hunk 2 above)
Comment (Arya274) [reply plugins/_afk.js:27]: Lumayan hehe

**Comment by Arya274** [comment]
#

**Comment by Arya274** [comment]
D

**Comment by Arya274** [comment]
let require('util')
let path = require('path')
let { spawn } = require('child_process')

// Magernulis By MFarelS:V
// Recode: Arya274
let handler  = async (m, { conn, text }) => {
  let d = new Date
  let tgl = d.toLocaleDateString('id-Id')
  let hari = d.toLocaleDateString('id-Id', { weekday: 'long' })
 text,
await conn.sendFile(m.chat, global.API('xteam', '/magernulis', { text, nama: conn.getName(m.sender), kelas: ' ' }, 'APIKEY'), 'nulis.jpg', 'Nahhh sudah jadi..\nSubsribe Youtube: *Drawl Nag*', m)
}
handler.help = ['magern'].map(v => v + 'ulis (teks)')
handler.tags = ['nulis']
handler.command = /^magernulis$/i
handler.owner = false
handler.mods = false
handler.premium = false
handler.group = false
handler.private = false

handler.admin = false
handler.botAdmin = false

handler.fail = null
handler.register = true

module.exports = handler

Sc magernulis ... Klo yg kedua nanti dulu

**Comment by Arya274** [comment]
:v

**Pull request closed without merging**